components/lane_follower/src/individual_follower.py

```diff
@@ -0,0 +1,172 @@
+# Lane Follower for a single camera
+# Applies a perspective transform to an image, and isolates lane lines
+import numpy as np
+import math
+import cv2
+
+
+class IndividualFollower:
+    # Constants for how we process images
+    # These should be parametrized and better defined at a later date: as part of testing
+    NWINDOWS = 9
+    SEARCH_WIDTH = 100
+    MINPIXELS = 40
+    LANE_POLY_SIZE = 2
+
+    # Rendering Parameters
+    BOX_COLOR = (0, 255, 0)
+    LANE_COLOR = (255, 0, 0)
+
+    DRAW_THICKNESS = 2
+
+    def __init__(self):
+        # self.fit is set in Plot_Line
+        self.fit = None
+        # These two are set below
+        self._binary_warped = None
+        self.histogram = None
+
+    # Need to determine if these should remain properties
+    # Or if they should be passed as arguments to Plot_Line
+    @property
+    def _binary_warped(self, value: np.ndarray):
+        self._binary_warped = value
+        self.histogram = np.sum(
+            self.binary_warped[self.binary_warped.shape[0] // 2 :, :], axis=0
+        )
```

Comment:
I'm not sure I follow why you're making this a property? If this is a getter, shouldn't it be a public member?